src/pages/Maps.tsx

```diff
@@ -1,20 +1,26 @@
-import { PointType } from "../types";
-import { useState } from "react";
+import { LocationMarker, PointType } from "../types";
+import { useEffect, useState } from "react";
 import { useFetchQuery } from "../hooks/useFetchQuery";
-import { Button } from "../components/Button";
-import { Map } from "./Map";
+import { Map } from "../components/Map";
 import { Section } from "../components/Section";
 
 export const Maps = () => {
   const { data, isLoading, error } = useFetchQuery<PointType>("point");
-  const [location, setLocation] = useState<{
-    coordinates: {
-      lat: number | null;
-      lng: number | null;
-    };
-    name: string | null;
-    zoom: number | null;
-  }>({ coordinates: { lat: 50.0755, lng: 14.4378 }, name: "Praha", zoom: 10 });
+  const [locationMarkers, setLocationMarkers] = useState<LocationMarker[]>([]);
+
+  useEffect(() => {
+    if (data) {
+      const transformedData = data.map((point) => ({
```

Comment:
Tohle tady je zbytecny, nepridava to v celku zadnou hodnotu, ale zvysuje to narocnost, ze pres to pole iterujes jeste po mimo ten return